src/libs/ValidationUtils.js

```diff
@@ -248,6 +248,31 @@ function getAgeRequirementError(date, minimumAge, maximumAge) {
     return ['privatePersonalDetails.error.dateShouldBeAfter', {dateString: format(longAgoDate, CONST.DATE.FNS_FORMAT_STRING)}];
 }
 
+/**
+ * Validate that given date is not expired more than two minutes.
+ *
+ * @param {String} inputDate
+ * @returns {String|Array}
+ */
+function isExpiredData(inputDate) {
```

Comment:
```suggestion
function getDatePassedError(inputDate) {
```
Something more descriptive like this? Also to indicate that an error is returned rather than a bool (which you might expect with an `is...` function name.

Reply:
Good idea, thanks!